client/src/components/Videos.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Rating from "./Rating";
+
+function Videos(props) {
```

Comment:
Since this component actually is for one video, it may be a better case if you just name this as `Video` instead of `Videos`. Not because it is wrong, but just for better understanding and not to confuse minds in the future.

Reply:
:+1: